.ko.yaml

```diff
@@ -1 +1 @@
-defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.20.3-bullseye.0
+defaultBaseImage: registry.k8s.io/build-image/go-runner:v2.3.1-go1.21.3-bullseye.0
```

Comment:
Does this image exist? Looks like it should be the `bookworm` variant: https://github.com/kubernetes/k8s.io/blob/d0222fb7613e8ea893025d92b1f93d22d10b88fa/registry.k8s.io/images/k8s-staging-build-image/images.yaml#L646

Reply:
My bad, updated the tags to be proper ones